### DISCUSSION TITLE
How to change color of navbar, and nav elements, logo

### DISCUSSION THREAD

**Discussion by checkifexist** [ansibleguy76/ansibleforms#154] [Q&A] [answered]
Can we change color of navbar, edit logo? if yes how

**Comment by ansibleguy76** [comment, marked as answer]
Logo is easy :
https://github.com/ansibleguy76/ansibleforms-docker/blob/main/docker-compose.yml
=> there is a comment about mapping the logo (it's basically overwriting the logo with your own) using a file mapping.

Navbar is harder, I'm using sass and sscs so the css is getting compiled.  Let me check if I can get that more parameterized.

**Comment by ansibleguy76** [comment]
version 6.x will have a dark mode, I'll see if I can somehow make at least the navbar color changeable.
